agora-test/README.org

```diff
@@ -0,0 +1,10 @@
+#+Title: Agora Test
+This folder is the test suite for Agora governance system.
+
+- =/Spec= contains different tests for different elements of Agora.
+- =/Spec/Model= contain =apropos-tx= model for logical suite
+  generation and tests.
+- =/Spec/Sample= contains primitive hand-made example values.
+- =Util.hs= contains helper functions
+
+Currently, planning migration to =plutarch-test= with golden test.
```

Comment:
The idea isn't to _replace_ our current tests with `plutarch-test`, but rather to complement them. `plutarch-test` should be used for the following:
- benchmarking
- unit tests
- goldens

Reply:
Other than that, this PR looks good to me.